config/routes.rb

```diff
@@ -3,11 +3,15 @@
   root 'welcome#index'
   resource :session
 
+  get '/login' => 'sessions#new'
+  get '/register' => 'users#new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.